source/frontend/src/hooks/pims-api/useApiAcquisitionFile.ts

```diff
@@ -42,6 +42,19 @@ export const useApiAcquisitionFile = () => {
             Accept: 'application/vnd.ms-excel',
           },
         }),
+      exportAcquisitionFiles: (
+        filter: IPaginateAcquisition,
+        outputFormat: 'csv' | 'excel' = 'excel',
+      ) =>
+        api.get<Blob>(
+          `/reports/acquisition?${filter ? queryString.stringify({ ...filter, all: true }) : ''}`,
+          {
+            responseType: 'blob',
+            headers: {
+              Accept: outputFormat === 'csv' ? 'text/csv' : 'application/vnd.ms-excel',
+            },
```

Comment:
csv format not supported by our export endpoints (currently)

Reply:
removed